www/src/data/sidebars/doc-links.yaml

```
@@ -454,6 +454,8 @@
               items:
                 - title: Building an E-commerce Site with Shopify
                   link: /docs/building-an-ecommerce-site-with-shopify/
+                - title: Process Payments with Stripe
+                  link: /docs/process-payments-with-stripe/
```

```suggestion
                  link: /docs/processing-payments-with-stripe/
```

I'd like to use the gerund here for consistency with other docs. This means updating the name of the Markdown file as well.